compiler-core/src/javascript/tests/numbers.rs

```diff
@@ -12,6 +12,14 @@ fn go() {
     0b00001111
     0o17
     0xF
+    0.01e-1
+    0.01e-0
+    -10.01e-1
+    -10.01e-0
+    100.001e523
+    -100.001e-523
+    100.001e123_456_789
+    -100.001e-123_456_789
```

Comment:
Could these go in a new test please? This one is already testing far too many things! Thanks

Reply:
yes, and I just noticed that I put them in the `int_literals` test by mistake.